packages/tools/devtools/devtools-core/src/Features.ts

```diff
@@ -71,9 +42,24 @@ export enum ContainerDevtoolsFeature {
  *
  * @internal
  */
-export type ContainerDevtoolsFeatureFlags = {
+export interface ContainerDevtoolsFeatureFlags {
+	/**
+	 * Indicates that the Container Devtools supports visualizing the data associated with the Container.
+	 */
+	"containerDataVisualization"?: boolean;
+
 	/**
-	 * Indicates whether or not a given {@link ContainerDevtoolsFeature} is supported by an instance of Devtools.
+	 * Indicates that the Container Devtools supports editing the data associated with the Container.
 	 */
-	[Feature in ContainerDevtoolsFeature]?: boolean;
-};
+	"containerDataEditing"?: boolean;
+
+	/**
+	 * Indicates that the Container Devtools supports visualizing the data associated with the Container.
+	 *
+	 * @deprecated
+	 *
+	 * This exists for backwards compatibility only.
+	 * Use {@link ContainerDevtoolsFeatureFlags.containerDataVisualization} instead.
+	 */
+	"container-data"?: boolean;
```

Comment:
Since the properties are getting inlined, I figured it would be better to replace this with a more accessible identifier.